api/schema.py

```diff
@@ -33,3 +38,41 @@ def _get_response_bodies(self, direction: str = "response") -> dict[str, Any]:
                 status_code="400",
             )
         return bodies
+
+
+class TeacherNonTeachingHelpProvidedFieldFix(OpenApiSerializerFieldExtension):
+    """
+    This class automatically extends schema for NonTeachingHelpSerializerField,
+    showing this object properly in docs.
+    """
+
+    target_class = "api.serializers.non_teaching_help.NonTeachingHelpSerializerField"
+
+    def map_serializer_field(
+        self, auto_schema: AutoSchema, direction: Literal["request", "response"]  # noqa: ARG002
```

Comment:
If you're really into avoiding repetitions, you could create 
```py
class Direction = Literal["request", "response"]
```
but it's totally up to you :)

Reply:
I think without `class` :) Done

Reply:
Yes, of course 🤦🏻